tsconfig.build.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What's the use of a second tsconfig? Why not add the include in tsconfig.json

Reply:
You are right, I was having problems with tslint but I have made some test now and I think we don't need this file.

Reply:
I've deleted the file in the last commit